database/database.go

```diff
@@ -0,0 +1,38 @@
+package database
+
+import (
+	"database/sql"
+	"fmt"
+	"time"
+
+	_ "github.com/lib/pq"
+)
+
+const (
+	host     = "localhost"
+	port     = 5432
+	user     = "dummyuser"
+	password = "12345678"
+	dbname   = "dummydb"
+
+	maxLifeTime  = 30 * time.Second
+	maxIdleTime  = 5 * time.Second
+	maxOpenConns = 5
+	maxIdleConns = 1
```

Comment:
I believe this should be configurable as well, so that you don't need to change the code if you want to change the numbers.